gatsby-node.js

```diff
@@ -466,11 +466,13 @@ exports.createSchemaCustomization = ({ actions }) => {
       field_stat_range: Boolean
       field_stat_value: String
       field_stat_value_end: String
+      field_font_awesome_icon: String
       relationships: paragraph__program_statisticRelationships
     }
     type paragraph__program_statisticRelationships implements Node {
       field_stat_icon: media__image @link(from: "field_stat_icon___NODE")
       field_stat_type: taxonomy_term__statistic_type @link(from: "field_stat_type___NODE")  
+      field_font_awesome_icon_colour: taxonomy_term__colour_variables @link(from: "field_font_awesome_icon_colour___NODE")

```

Comment:
Wondering why this is here when the font awesome icon colour is controlled via CSS? It's not supposed to be set for Stats in the Drupal backend